src/Schedule.h

```diff
@@ -100,6 +100,32 @@ enum class TailStrategy {
      * instead of a multiple of the split factor as with RoundUp. */
     ShiftInwards,
 
+    /** Equivalent to ShiftInwards, but protects values that would be
```

Comment:
I really think we need to have some kind of tutorial for all of the different variants of TailStrategy: there are a lot of them and some of them are pretty subtle, so it's not really obvious how exactly they work. 

No need to block PR on this though, but maybe would be good to create an issue.